src/hotspot/share/opto/parse1.cpp

```diff
@@ -1575,6 +1575,7 @@ void Parse::do_one_block() {
 #endif //ASSERT
 
     do_one_bytecode();
+    if (C->failing()) return;
```

Comment:
I think you can simple call `failing` here and potentially at other places as well.

Reply:
Okay. Have not found any other places where that is possible though.